roles/schema_registry/tasks/health_check.yml

```diff
@@ -14,7 +24,27 @@
   until: result1.status == 200
   retries: 30
   delay: 5
-  when: schema_registry_authentication_type != 'mtls'
```

Comment:
should the value of sr_authentication_type variable also take the value bearer in case of oauth?

Reply:
no, this is authentication, bearer is for authorization

Reply:
isnt it doing both authentication and authorization

Reply:
no, this is only for authentication, for authorization we need rbac on cluster. This task will be run on non rbac clusters having mTLS authentication

Reply:
oh my bad i asked this question at wrong place. I meant to ask it for the task below where mtls is false and auth is ldap or oauth.

Reply:
ah got it, bearer is only for authorization. For authentication it depends on what the customer has defined [here](https://github.com/confluentinc/cp-ansible/blob/7.6.1-post/roles/variables/defaults/main.yml#L733)